rules/rules.libsonnet

```diff
@@ -58,6 +58,37 @@
           },
         ],
       },
+      {
+        name: 'kube-scheduler.rules',
+        rules: [
+          {
+            record: 'cluster:%s:quantile' % metric,
```

Comment:
Should this be `cluster_quantile:%s:`?  Labels go first according to https://prometheus.io/docs/practices/rules/.

Although I guess `cluster_quantile:scheduler_e2e_scheduling_latency:histogram_quantile` is a bit of a mouth full.